notebooks/04_model_select_and_optimize/aml_hyperparameter_tuning.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>I would remove the mention to databricks, here we are not using it. Also, it's dangerous to say things like, this is not supported for now, because, in a month this feature can be added and we will forgot that we wrote this line here</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Make sense. Removed.